stampit.js

```diff
@@ -164,9 +176,13 @@ compose = function compose(factories) {
       if (source.fixed.enclose) {
         f.enclose = f.enclose.concat(source.fixed.enclose);
       }
+
+      if (source.fixed.statics) {
+        f.statics = mixInChain(f.statics, source.fixed.statics);
```

Comment:
This will mix two objects *including* their prototype chain. Was that intentionally? I feel like `mixIn` would be safer and more predictable behavior.